src/app/components/member/member-profile/member-profile.component.html

```diff
@@ -1,60 +1,86 @@
-   <div class="image-container">
-      <img class="img-profile rounded-circle" src="https://source.unsplash.com/QAB-WJcbgJk/60x60">
-       </div>
-                            
-      <div class="ml-auto">
-<input type="button" class="btn btn-primary d-none" id="btnDiscard" value="Discard Changes" />
-     </div>
-     </div>
-    </div> 
-
- <div class="row">
-<div class="col-12">
-  <ul class="nav nav-tabs mb-4" id="myTab" role="tablist">
- <li class="nav-item">
-<a class="nav-link active" id="basicInfo-tab" data-toggle="tab" href="#basicInfo" role="tab" aria-controls="basicInfo" aria-selected="true">Basic Info</a>
- </li>
- <li class="nav-item">
- <a class="nav-link" id="connectedServices-tab" data-toggle="tab" href="#connectedServices" role="tab" aria-controls="connectedServices" aria-selected="false">Connected Services</a>
-</li>
-</ul>
-  <div class="tab-content ml-1" id="myTabContent">
-   <div class="tab-pane fade show active" id="basicInfo" role="tabpanel" aria-labelledby="basicInfo-tab">
-                                    
- <div class="row">
-          <div class="col-sm-3 col-md-2 col-5">
-<label style="font-weight:bold;">First Name</label>
-</div>
-<div class="col-md-8 col-6">{{firstname}}  </div>
- </div>
-    <hr />
+  <div class="row">
+    <div class="col-12">
+      <div class="card">
+        <div class="card-body">
+          <div class="card-title mb-4">
+            <div class="d-flex justify-content-start">
+              <div class="image-container">
+                <img class="img-profile rounded-circle" src="https://source.unsplash.com/QAB-WJcbgJk/60x60">
+              </div>
+              <div class="ml-auto">
+                <input type="button" class="btn btn-primary d-none" id="btnDiscard" value="Discard Changes" />
+              </div>
+            </div>
+          </div>
+          <div class="row">
+            <div class="col-12">
+              <ul class="nav nav-tabs mb-4" id="myTab" role="tablist">
+                <li class="nav-item">
+                  <a class="nav-link active" id="basicInfo-tab" data-toggle="tab" href="#basicInfo" role="tab"
+                    aria-controls="basicInfo" aria-selected="true">Basic Info</a>
+                </li>
+                <li class="nav-item">
+                  <a class="nav-link" id="connectedServices-tab" data-toggle="tab" href="#connectedServices" role="tab"
+                    aria-controls="connectedServices" aria-selected="false">Connected Services</a>
+                </li>
+              </ul>
+              <div class="tab-content ml-1" id="myTabContent">
+                <div class="tab-pane fade show active" id="basicInfo" role="tabpanel" aria-labelledby="basicInfo-tab">
+                  <div class="row">
+                    <div class="col-sm-3 col-md-2 col-5">
+                      <label style="font-weight:bold;">ID</label>
+                    </div>
+                    <div class="col-md-8 col-6">{{member.id}} </div>
```

Comment:
ID is used for internal (code) purposes we don't usually show the id of the member as it adds nothing to the viewer. So please take out the id part.